example.toml

```diff
@@ -134,11 +134,23 @@ meme = '^\S*$'
 # moderator privileges.
 # Unlike most strings, these are not expanded with environment variables.
 privileges = [
-	{ name = 'zephyrtronium', level = 'moderator' },
+    { name = 'zephyrtronium', level = 'moderator' },
 ]
 
 [twitch.bocchi.emotes]
 'btw make sure to stretch, hydrate, and take care of yourself <3' = 1
 
 [twitch.bocchi.effects]
 'AAAAA' = 44444
+
+[discord]
+# Discord bot token, requires the Message Content intent enabled.
+token = ''
+# Server configurations, each server/guild must have a config section here
+# this is because we may want to learn from both twitch and discord
+# however only use discord messages in discord due to how pings are
+# formatted, emotes, etc...
+# Other parameters work the same way as the twitch configurations.
+configs = [
```

Comment:
Instead of an inline array, this should either use `[[discord.configs]]` syntax or work the same way as Twitch channel config.